### PR TITLE
SLLS-223: Slack notification on pipeline failure

### PR DESCRIPTION
**Pull request by thahnen**
## INFO: This will be merged AFTER the next release, just in case to not break anything.

### CODE REVIEW

File: .cirrus/slack-notification.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -euo pipefail
+
+curl -X POST https://slack.com/api/chat.postMessage \
+  -H "Authorization: Bearer ${SLACK_TOKEN}" \
+  -H 'Content-type: application/json; charset=utf-8' \
+  --data-binary @- <<EOF
+{
+  "channel": "team-sonarlint-vscode",
```
Comment (sophio-japharidze-sonarsource) [inline .cirrus/slack-notification.sh:10]: `squad-ide-vscode` or `squad-ide-vscode-bots` ?

Comment (thahnen) [reply .cirrus/slack-notification.sh:10]: @sophio-japharidze-sonarsource I copy pasted it from the release workflow (to always have the "correct" one), maybe this is incorrect as well. I can change both if you like 😄

Comment (sophio-japharidze-sonarsource) [reply .cirrus/slack-notification.sh:10]: I guess it also got outdated after the renaming, we haven't released yet since then to see it fail :D

Comment (thahnen) [reply .cirrus/slack-notification.sh]: Done!

**Review by sophio-japharidze-sonarsource — APPROVED** [review]

**Pull request merged**